### PR TITLE
Add global styling

### LINKED ISSUE #4: Add global styling
## ToDos

- [x] import fonts
- [x] set variables for colors
- [x] import global style in storybook
- [x] import global style in app.js

### PR DESCRIPTION
**Pull request by Frenzy265**
close #4

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -10,6 +10,14 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link
```
Comment (slawoe) [inline client/public/index.html:13]: Just a hint and not really relevant for your project: maybe import the fonts as assets. Cause directly from google fonts your page will have Google cookies - this is unnecessary. You can import them then from your assets directly to your GlobalStyles.

**Review by slawoe — APPROVED** [review]
Looks good to me.
Nice job so far. 

Just a tip for importing your fonts (see my comment).

File: client/src/globalstyle.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    --action-color: #F2CB05;
+    --noaction-color: #fff;
+    --light-font-color: #fff;
+    --dark-font-color: var(--main-color)
+}
+
+body {
+    margin: 0;
+    background: var(--main-color);
+    font-family: "Roboto", sans-serif;
```
Comment (AlexisRoe) [inline client/src/globalstyle.js]: you can also do this in variables, this mabye helps you in the future in other components ;-)

f.e. --font-body: "Roboto", sans-serif;

**Review by AlexisRoe — COMMENTED** [review]
good job

File: .prettierignore [hunk 3]
```diff
@@ -1,4 +1,5 @@
 # Ignore artifacts:
 build
 coverage
-storybook-static
\ No newline at end of file
+storybook-static
+ISSUE_TEMPLATE
```
Comment (lmachens) [inline .prettierignore]: The ISSUE_TEMPLATE should be uploaded to GitHub, otherwise it won't work :).

**Review by lmachens — COMMENTED** [review]
gj

File: client/public/index.html (hunk 1 above)
Comment (Frenzy265) [reply client/public/index.html:13]: Thanks for your hint! 👍🏻

File: client/src/globalstyle.js (hunk 2 above)
Comment (Frenzy265) [reply client/src/globalstyle.js]: Good idea! I will do this! 😉

File: .prettierignore (hunk 3 above)
Comment (Frenzy265) [reply .prettierignore]: THX! I solved the problem and have prettify the issue template :-)

**Pull request merged**